src/main/groovy/de/gliderpilot/gradle/semanticrelease/SemanticReleaseChangeLogService.groovy

```diff
@@ -237,6 +242,7 @@ class SemanticReleaseChangeLogService {
 
         Release release = repo.releases().create(tag)
         new Release.Smart(release).body(changeLog(commits(Version.valueOf(version.previousVersion)), version).toString())
+        releaseAsset(release.assets(), "$version.version")
```

Comment:
The second parameter is named `currentTag`, but the instead the version is passed. The question is, if version and/or tag are necessary at all. The sample code in README.md could also be written as

```
assets.upload(tasks.jar.outputs.files.singleFile.bytes, "application/zip", tasks.jar.archiveName)
```

Alternatively, the closure could be named `releaseAssets` (plural) instead, without parameters, expected to return a List of files. And instead of calling `releaseAsset(release.assets())`, it would be

```
releaseAssets().each { release.assets().upload(it.bytes, URLConnection.guessContentTypeFromName(it.name), it.name) }
```

with 

```
semanticRelease {
    changeLog {
        releaseAssets = {
            tasks.jar.outputs.files
        }
    }
}
```

But that is less powerful. What do you think?